docs/consume-packages/Dependency-Resolution.md

```diff
@@ -19,7 +19,9 @@ When multiple packages have the same dependency, then the same package ID can ap
 
 When installing packages into projects using the PackageReference format, NuGet adds references to a flat package graph in the appropriate file and resolves conflicts ahead of time. This process is referred to as *transitive restore*. Reinstalling or restoring packages is then a process of downloading the packages listed in the graph, resulting in faster and more predictable builds. You can also take advantage of wildcard (floating) versions, such as 2.8.\*, avoiding expensive and error prone calls to `nuget update` on the client machines and build servers.
 
-When the NuGet restore process runs prior to a build, it resolves dependencies first in memory, then writes the resulting graph to a file called `project.assets.json` in the `obj` folder of a project using PackageReference. MSBuild then reads this file and translates it into a set of folders where potential references can be found, and then adds them to the project tree in memory.
+When the NuGet restore process runs prior to a build, it resolves dependencies first in memory, then writes the resulting graph to a file called `project.assets.json`. 
+The assets file is located at `MSBuildProjectExtensionsPath`, which defaults to the 'obj' of the project. 
```

Comment:
`which defaults to the 'obj' of the project` -> `which defaults to the project's 'obj' folder`?

Reply:
It's not super clear that `MSBuildProjectExtensionsPath` is an MSBuild variable. Could this be clarified?

Reply:
The MSBuild part wasn't clear enough? :D

Regardless, I want to link the known msbuild properties page like https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2017#list-of-common-properties-and-parameters, but I'm not 100% sure how we usually link to page like that (do we use the full link etc).